backend/package.json

```diff
@@ -66,6 +66,7 @@
     "graphql-tag": "~2.10.3",
     "helmet": "~3.22.0",
     "ioredis": "^4.16.1",
+    "imgproxy": "^0.1.2",
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-01T06:37:05Z" title="Wednesday, April 1st 2020, 8:37:05 am +02:00">Apr 1, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -65,6 +65,7 @@
     "graphql-shield": "~7.2.1",
     "graphql-tag": "~2.10.3",
     "helmet": "~3.22.0",
+    "imgproxy": "^0.1.2",
```


from what I've seen about this package, I think you made a great choice @roschaefer !!
it has thousands of stars, is maintained, and fits perfectly what we want to do!!